messaging-experiments.yaml

```diff
@@ -1,11 +1,12 @@
-- id: AW_PULL_FACTOR_PRIVACY_1
+- id: AW_PULL_FACTOR_PRIVACY_2
   enabled: true
   filter_expression: ('76' in env.version) || (localeLanguageCode == "en" &&
-    firefoxVersion >= 76 && [userId, "aw_pull_factor_privacy_1"]|bucketSample(0,
+    firefoxVersion >= 76 && [userId, "aw_pull_factor_privacy_2"]|bucketSample(0,
     2000, 10000) &&
-    (!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue) || 'aw_pull_factor_privacy_1' in activeExperiments))
+    !(((currentDate|date - profileAgeCreated) / 86400000) > 1) &&
```

Comment:
Filter expression here will exclude all users from enrolling whose profile age is more than 1 day. 

I believe its a narrower filtering than Normandy recipe where we are enrolling profiles whose profile creation date is more than experiment start date
(!(normandy.telemetry.main.environment.profile.creationDate < (\"2020-04-14\"|date / 1000 / 60 / 60 / 24))\n

https://normandy.cdn.mozilla.net/api/v3/recipe/942/

Reply:
Do we want to keep `!('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)`?

Reply:
I don't see any harm in keeping it, a new user shouldn't have seen about:welcome and should be evaluated to true